diffs/20231115-Ethereum-0xC13e21B648A5Ee794902342038FF3aDAB66BE987-pre_20231115-Ethereum-0xC13e21B648A5Ee794902342038FF3aDAB66BE987-post.md

```diff
@@ -0,0 +1,136 @@
+## Reserve changes
+
+### Reserve altered
+
+#### WBTC ([0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599](https://etherscan.io/address/0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599))
+
+| description | value before | value after |
+| --- | --- | --- |
+| isFrozen | true | false |
+| supplyCap | 1,000 WBTC | 3,000 WBTC |
+| borrowCap | 500 WBTC | 2,000 WBTC |
+| liquidationBonus | 6.25 % | 7 % |
+| interestRateStrategy | [0xf2812d7a07573322D4Db3C31239C837081D8294E](https://etherscan.io/address/0xf2812d7a07573322D4Db3C31239C837081D8294E) | [0xD2139d6d63Acb1e7Cc91cE32bbD86eFb17eBEe46](https://etherscan.io/address/0xD2139d6d63Acb1e7Cc91cE32bbD86eFb17eBEe46) |
+| optimalUsageRatio | 65 % | 60 % |
+| maxExcessUsageRatio | 35 % | 40 % |
```

Comment:
Why is this getting updated?

Reply:
Max excess usage ratio is `100% - optimalUsageRatio`.